docs/howto/mbed-tools-howto.md

```diff
@@ -1,254 +1 @@
-    ```
-1. To prepare the Mbed configuration information for use with a specific target and toolchain, navigate to the project's root folder and run:
-
-    ```
-    mbed-tools configure -m <target> -t <toolchain>
-    ```
-
-    - The supported targets are `K64F`, `DISCO_L475VG_IOT01A`, `NRF52840_DK`
-    - The supported toolchains are `GCC_ARM` and `ARM`.
-
-    Example for FRDM-K64F and GCC:
-
-    ```
-    mbed-tools configure -m K64F -t GCC_ARM
-    mbed_config.cmake has been generated and written to '/Users/UserName/Development/Blinky/.mbedbuild'
-    ```
-
-## Build the project
-
-Currently, the new Cmake system supports three boards: `K64F`, `DISCO_L475VG_IOT01A` and `NRF52840_DK`.
-
-Use CMake to build your application:
-
-1. Navigate to the project's root folder.
-1. Set the build parameters:
-
-    ```
-    cmake -S . -B cmake_build -GNinja -DCMAKE_BUILD_TYPE=<profile>
-    ```
-    - -S <path-to-source>: Path to the root directory of the CMake project. We use `.` to indicate we're building from the current directory.<!--at no point until now did we tell them to navigate to the directory, though-->
-    - -B <path-to-build>: Path to the build output directory. If the directory doesn't already exist, CMake will create it. We use `cmake_build` as the output directory name; you can use a different name.
-    - -GNinja: To use the Ninja tool.
-    - -DCMAKE_BUILD_TYPE: Build type. The value (`profile`) can be `release`, `debug` or `develop`.
-
-1. Build:
-
-    ```
-    cmake --build cmake_build
-    ```
-
-    This generates two files in the build output directory (`cmake_build` in this example): HEX and BIN.
-
-1. Drag and drop the generated file to your board.
-
-
-## Configure and build in a single step
-
-You can use a single command to configure (select your target and toolchain) and build the project at once:
-
-1. To build and configure:
-
-    ```
-    mbed-tools build -m <target> -t <toolchain>
-    ```
-    - -t: The toolchain you are using to build your app
-    - -m: A build target for an Mbed-enabled device
-
-    Example for FRDM-K64F and GCC:
-
-    ```
-    mbed-tools build -m K64F -t GCC_ARM
-    ```
-
-1. To perform an iterative build on previously configured target:
-
-    ```
-    mbed-tools build
-    ```
-
-## List library dependencies of an application
-
-All library dependencies can be listed with their URL and Git references as follows:
-
-```
-mbed-tools libs
-```
-
-## Logging
-
-To specify the log level, use the verbose logging option (`-v`) before the first argument.
-
-If you don't use `-v`, by default the log will show only errors. These are the following log levels:
-
-- `-v`: Warning and errors.
-- `-vv`: Information, warning and errors.
-- `-vvv`: Debug, information, warning and errors.
-
-For example:
-
-```
-mbed-tools -vv configure -m <target> -t <toolchain>
-```
+This document has moved to: https://os.mbed.com/docs/mbed-os/v6.5/build-tools/mbed-cli-2.html
```

Comment:
We should use the `latest` instead of hard-coded versions. But currently, https://os.mbed.com/docs/mbed-os/latest points to https://os.mbed.com/docs/mbed-os/v6.4/introduction/index.html. Should we wait till 6.5 docs is released?

Reply:
The URL might change again for 6.6...